.github/workflows/ci.yml

```
@@ -272,7 +272,7 @@ jobs:
           ref: ${{ matrix.nextcloudVersion }}
 
       - name: Setup PHP ${{ format('{0}.{1}', matrix.phpVersionMajor,matrix.phpVersionMinor) }}
-        uses: shivammathur/setup-php@v2
+        uses: shivammathur/setup-php@verbose
```

is this change needed?
```suggestion
        uses: shivammathur/setup-php@v2
```

---

forgot to revert it thanks.